R/status.R

```diff
@@ -283,8 +283,19 @@ renv_status_check_synchronized <- function(project,
 
   }
 
-  # */not recorded/not used ----------------------------------------------------
-  # No action; it's okay if some auxiliary packages are installed.
+  # installed/*/not used -------------------------------------------------------
+  records <- library %>%
+    exclude(packages) %>%
+    exclude(renv_packages_recommended())
+
+  if (length(records)) {
+    renv_pretty_print_records(
+      records,
+      preamble = "The following packages are installed but not used."
+    )
```

Comment:
There's no postamble, because there at least four things you might want to do:

* Nothing, because you're using the packages interactively and they don't need to be inclued.
* You might want to run `clean()` to get rid of them
* You might want to add them to `_deps.R` to track them
* You might want to change the snapshot type